libs/core/src/lib/action-sheet/action-sheet-control/action-sheet-control.component.ts

```diff
@@ -36,4 +37,12 @@ export class ActionSheetControlComponent {
     onClick(): void {
         this.clicked.emit();
     }
+
+    /** @hidden */
+    _focus(): void {
+        this._elementRef.nativeElement.firstChild.focus();
```

Comment:
actually there are no limitations for the action sheet control content hence `firstChild` may be the not focusable element.

Reply:
@platon-rov  added logic to make element focusable if not.